--- gnome-boxes-3.38.1/src/meson.build.orig	2020-10-06 15:59:46.898945600 +0200
+++ gnome-boxes-3.38.1/src/meson.build	2020-10-06 21:31:44.973174909 +0200
@@ -175,16 +175,7 @@
     dependency ('gudev-1.0', version: '>= 165'),
   ]
 
-  libhandy = subproject(
-    'libhandy',
-    default_options: [
-      'package_subdir=' + meson.project_name(),
-      'examples=false',
-      'glade_catalog=disabled',
-      'tests=false',
-    ]
-  )
-  dependencies += libhandy.get_variable('libhandy_vapi')
+  dependencies += dependency ('libhandy-0.0', version: '>= 0.0.11')
 
   if get_option('rdp')
     gtk_frdp = subproject(
